bench/app.py

```diff
@@ -909,8 +909,15 @@ def install_app(
 
 	app_path = os.path.realpath(os.path.join(bench_path, "apps", app))
 
+	env = {}
+	# macOS needs a custom PKG_CONFIG_DIR for frappe
+	if app == "frappe" and sys.platform == "darwin":
+		env = {
+			"PKG_CONFIG_PATH": "/opt/homebrew/opt/mariadb-connector-c/lib/pkgconfig",
```

Comment:
This looks brittle

Maybe something like `pkg-config --variable=pc_path mariadb-connector-c` or `$(brew --prefix mariadb-connector-c)/lib/pkgconfig` and ask user to set it manually if envvar is not set properly

Reply:
Hmm yeah my previous version used brew --prefix mariadb-connector-c to get it dynamically, had done this to quickly test something while I had access to a Mac, will re-add that part with this logic soon.